test/expected/issue_50_delete_filters.out

```diff
@@ -0,0 +1,170 @@
+select 1 from pg_create_logical_replication_slot('realtime', 'wal2json', false);
+ ?column? 
+----------
+        1
+(1 row)
+
+create table public.notes(
+    id int primary key,
+    body text
+);
+insert into realtime.subscription(subscription_id, entity, claims, filters)
+select
+    seed_uuid(id),
+    'public.notes',
+    jsonb_build_object(
+        'role', 'authenticated',
+        'email', 'example@example.com',
+        'sub', seed_uuid(id)::text
+    ),
+    array[(column_name, op, value)::realtime.user_defined_filter]
+from
+    (
+        values
+            (1 , 'body', 'eq', 'bbb'),
+            (2 , 'id', 'eq', '2')
+    ) f(id, column_name, op, value);
+select subscription_id, filters from realtime.subscription;
+           subscription_id            |      filters      
+--------------------------------------+-------------------
+ f4539ebe-c779-5788-bbc1-2421ffaa8954 | {"(body,eq,bbb)"}
+ 5211e8ec-8c25-5c7f-9b03-6ff1eac0159e | {"(id,eq,2)"}
+(2 rows)
+
+----------------------------------------------------------------------------------------
+-- When Replica Identity is Not Full, only filters referencing the pkey are respected --
+----------------------------------------------------------------------------------------
+insert into public.notes(id, body)
+values
+    (1, 'bbb'),
+    (2, 'ccc');
+select clear_wal();
+ clear_wal 
+-----------
+ 
+(1 row)
+
+delete from public.notes;
+select
+    rec,
+    is_rls_enabled,
+    subscription_ids,
+    errors
+from
+   walrus;
+                        rec                         | is_rls_enabled |            subscription_ids            | errors 
+----------------------------------------------------+----------------+----------------------------------------+--------
+ {                                                 +| f              | {}                                     | {}
```

Comment:
no `subscription_ids` present because there is a filter on `body` which is not part of the replica identity / not part of the primary key